src/components/intersection-observer/intersection-observer.spec.js

```diff
@@ -0,0 +1,88 @@
+import { expect } from 'chai';
+import { shallowMount } from '@vue/test-utils';
+import IntersectionObserverComponent from './intersection-observer.vue';
+
+describe('Intersection-Observer', function() {
+    it.skip('Component is an object', () => {
+        expect(IntersectionObserverComponent).to.be.an('object');
+        expect(IntersectionObserverComponent).to.be.not.empty;
+    });
+
+    it.skip('Component registers an observer', () => {
+        const wrapper = shallowMount(IntersectionObserverComponent, {});
+
+        const { observer, observerOptions } = wrapper.find(IntersectionObserverComponent).vm;
+
+        expect(observer).to.be.an('object');
+        expect(observer).to.be.not.empty;
+
+        expect(observerOptions).to.be.an('object');
+        expect(observerOptions).to.be.not.empty;
+    });
+
+    it('interserct-enter triggers on appearance', done => {
+        shallowMount({
+            components: {
+                ioc: IntersectionObserverComponent,
+            },
+            data() {
+                return {
+                    inlineStyle: {
+                        width: `${ window.innerWidth }px`,
+                        height: `${ window.innerHeight * 2 }px`,
+                    },
+                    isVisible: true,
+                };
+            },
+            methods: {
+                onVisible() {
+                    done();
```

Comment:
please remember to unmount/destroy your components once the tests are done. We don't want zombie views attached to the DOM generating side effects. Thanks